static/app/routes.tsx

```diff
@@ -39,11 +40,12 @@ const SafeLazyLoad = errorHandler(LazyLoad);
 export function makeLazyloadComponent<C extends React.ComponentType<any>>(
   resolve: () => Promise<{default: C}>
 ) {
+  const LazyComponent = lazy<C>(() => retryableImport(resolve));
```

Comment:
pulling this outside the RouteLazyLoad ensures that as long as makeLazyloadComponent isn't called multiple times this should be stable